og_ui/src/Controller/PeopleController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\og_ui\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Form\FormState;
+use Drupal\Core\Link;
+use Drupal\Core\Render\Element\Dropbutton;
+use Drupal\Core\Session\SessionManager;
+use Drupal\Core\Url;
+use Drupal\og_ui\Form\OgUiAddPeopleForm;
+use Drupal\og_ui\OgUi;
+use Drupal\og_ui\OgUiAdminRouteInterface;
+use Drupal\user\PrivateTempStoreFactory;
+use Drupal\views\Views;
+
+class PeopleController extends ControllerBase {
+
+  public static function access() {
+    /** @var OgUiAdminRouteInterface $plugin */
+    $plugin = OgUi::getGroupAdminPlugins()->createInstance(
+      \Drupal::routeMatch()->getRouteObject()->getRequirement('_plugin_id')
+    );
+
+    return $plugin
+      ->setGroup(OgUi::getEntity())
+      ->access();
+  }
+
+  /**
+   * Display list of people which belong to the group.
+   */
+  public function PeopleList() {
+    /** @var PrivateTempStoreFactory $session_storage */
+    $session_storage = \Drupal::service('user.private_tempstore');
```

Comment:
Used to store the last route we been in order to redirect the user back to where he came from.

We did the same thing in the message module.